meerkat/datapanel.py

```diff
@@ -817,3 +820,29 @@ def _copy_data(self) -> object:
 
     def __finalize__(self, *args, **kwargs):
         return self
+
```

Comment:
Let's add a method `groupby` so that we can call ```dp.groupby("a")```, this'll be just a simple wrapper around the main groupby function

Reply:
Resolved and tested